packages/material-ui/src/Radio/Radio.test.js

```diff
@@ -40,6 +41,21 @@ describe('<Radio />', () => {
     });
   });
 
+  describe('with icon', () => {
+    describe('with size small', () => {
+      it('should have the same fontSize with and without icon', () => {
+        const { getAllByTestId } = render(
+          <React.Fragment>
+            <Radio size="small" />
+            <Radio size="small" icon={<RadioButtonUncheckedIcon />} />
+          </React.Fragment>,
+        );
+        expect(getAllByTestId('RadioButtonUncheckedIcon')[0]).to.have.style('fontSize', '1.25rem');
+        expect(getAllByTestId('RadioButtonUncheckedIcon')[1]).to.have.style('fontSize', '1.25rem');
```

Comment:
A visual regression test is better here since browsers compute these styles anyway. The inline style is irrelevant and can has different values depending on the browser. This is why `test_browser` is failing.

For a visual regression test you just want to add a file with the rendered UI of this test to `test/regressions/fixtures//Radio`. For more information about our visual regression tst suite, check out `test/README.md`